File: Python PowerPoint Examples/TextApiDemo/replacePresentationText.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.text_api.TextApi(configuration)

#support file format: .pptx/.ppt/.pptm/.pps/.ppsx/.pot/.potx/.odp/
fileName = "ReplacePresentationText.pptx"
password = ""
storage = ""
inputFolder = "input"
oldValue = "r1C1"
newValue = "newValue"
#support :flase/true
ignore_case = "false"
result =api.replace_presentation_text(fileName, oldValue, newValue, password=password,storage=storage, folder=inputFolder, ignore_case=ignore_case)

File: JavaScript PowerPoint Examples/TextApiDemo/replacePresentationText.js
(function (){
    var appId = "your ID";
    var appKey = "your Key";
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudPowerpoint = require('../../src/index');
    var apiClient = new SpirecloudPowerpoint.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudPowerpoint.TextApi(apiClient);
    
    var name = "ReplacePresentationText.pptx";
    var folder = "input";
    var oldValue = "r1C1";
    var newValue = "newValue";
    var opt = { 'ignoreCase': 'false', 'folder': folder };
    var response = instance.replacePresentationText(name, oldValue, newValue, opt,
        function (error, data, response) {
            if (error) throw error;  
        });
})();